### PR TITLE
Tag master with the PR id that merged at that point.

### PR DESCRIPTION
**Pull request by jgraham**
For the manifest download stuff we have to have tags per PR merge on
master anyway, and this seems like rather useful information.

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/8005)

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-10-30 11:27:56
Finished: 2017-10-30 11:42:50





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20011)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/294805956)

**Comment by gsnedders** [comment]
Surely we need to authenticate to be able to add a tag?

**Comment by jgraham** [comment]
Oh, right we are authenticated for the deploy, but not at this point

**Comment by jgraham** [comment]
Something like that, perhaps

File: tools/ci/tag_master.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+import os
+import sys
+import urllib2
+
+here = os.path.abspath(os.path.dirname(__file__))
+wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+
+if not(wpt_root) in sys.path:
```
Comment (gsnedders) [inline tools/ci/tag_master.py]: This is pretty weird. `not(str)` is always `False`, and `False` will never appear in `sys.path`.

**Review by gsnedders — COMMENTED** [review]
See the below comment.

I'd also like to see some testing of this prior to landing, given prior CI changes breaking everything. Maybe change the branch that it tags for to `tag_master` and add a prefix to the tag?

**Review by gsnedders — APPROVED** [review]

**Pull request merged**